qute.jdt/com.redhat.qute.jdt.test/src/main/java/com/redhat/qute/jdt/template/TemplateGetDataModelProjectTest.java

```diff
@@ -267,6 +268,12 @@ private void testValueResolversFomInject(List<ValueResolverInfo> resolvers) {
 
 	}
 
+	private static void testValueResolversFromTemplateData(List<ValueResolverInfo> resolvers) {
+		assertValueResolver("org_acme_qute_ItemWithTemplateData",
+				"staticMethod(item : org.acme.qute.Item) : java.math.BigDecimal", "org.acme.qute.ItemWithTemplateData",
+				resolvers);
+	}

```

Comment:
You have a test with static method with namespace, that's nice, but please add test for:

 * static field without namespace. You could add a static field in ItemWithTemplateData which have no namespace.
 * static field with namespace by using your Statuses class (FOO:ON).
 * static method with namespace. You could add a static method in your Statuses which have FOO as namespace.